app/pages/_app.tsx

```diff
@@ -37,10 +36,6 @@ export default function App({ Component, pageProps }: AppProps) {
     i18n.activate(locale);
   }
 
-  useEffect(() => {
-    document.querySelector("html")?.setAttribute("lang", locale);
-  }, [locale]);
```

Comment:
Is it really necessary to remove this ? I am not sure this should be removed as it could be used somewhere else (maybe browser extensions also rely on this to propose translation to the user).

Reply:
It's being set automatically by Next.js + i18n